src/spec/base.ts

```diff
@@ -145,6 +145,11 @@ export interface BaseViewBackground
    * __Default value:__ `"#ddd"`
    */
   stroke?: Color | null;
+
+  /**
+   * The mouse cursor used over the view. Any valid [CSS cursor type](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor#Values) can be used.
+   */
+  cursor?: Cursor;
```

Comment:
Are we getting a default from Vega here? Are the Vega typings correct? Could you check?

Reply:
The vega-typings are correct, yes:
https://github.com/vega/vega/blob/master/packages/vega-typings/types/spec/config.d.ts#L309-L345

It's an enumeration of the valid strings.

Not sure what you mean with default from Vega. The cursor doesn't change by adding the property at least.

Reply:
We import the Vega typings in many places so often it's not necessary to override property types.

Reply:
There is a cursor definition in `BaseMarkConfig` from which this interface extends, but the documentation would say the property specifies the mouse cursor used over the _**mark**_.